SwiftUIDZ/SwiftUIDZ/ContentView.swift

```diff
@@ -8,15 +8,144 @@
 import SwiftUI
 
 struct ContentView: View {
+    
+    // MARK: - Private Constants
+    private enum Constants {
+        static let walletText = "Кошелек"
+        static let profileText = "Профиль"
+        static let currentBalanceText = "Текущий баланс:"
+        static let showSettings = "Показать настройки"
+        static let rub = "рублей"
+        static let refillTheBalanceButtonTitle = "Пополнить"
+        static let actionSheetRefillTheBalanceText = "Баланс пополнен"
+        static let actionSheetWriteOffFundsText = "Деньги списаны"
+        static let actionSheetInsufficientFunds = "Недостаточно средств"
+        static let emptyString = ""
+        static let openTwoCheck = "Второй счет открыт"
+        static let openCheckText = "Открыть счет"
```

Comment:
```suggestion
        static let openCheckText = "Открыть счет"
```
Check плохое слово для обозначения банковского счета